src/Views/FirmwareView.vala

```diff
@@ -120,12 +119,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
-            placeholder_alert_view.description = _("Firmware updates are not supported on this or any connected devices.");
+            placeholder_alert_view.title = _("No new firmware found");
+            placeholder_alert_view.description = _("Nothing on or connected to your system supports installing new firmware this way.");
```

Comment:
I don't think this is a better string than what is already there. It doesn't seem to clarify anything and instead adds a new question of where else firmware updates can be installed from

Reply:
It denotes other venues of upgrading firmware outside of the tool are viable, as it should.
(Ideally the tool should alert you to this if need be, but that is something for fwupd to fix.)
Unfortunately there is no peace of mind in relying on only one solution here. It is also unsolvable, so that is going to remain the case.

Newer firmware being available does not depend on fwupd being able to find or upgrade it, so the statement is factually untrue.

Reply:
It's not within the scope of this UI to tell people to sideload their own firmware, and that's probably not advisable for most people.

The UI is referring to the systems ability to update firmware and that firmware updates through the system are not supported. This is why it is correct and important that we are talking about firmware updates and not just "new firmware"

Reply:
Neither option tells anyone to install firmware manually.
The original says firmware "updates" aren't available, because they aren't supported, which is a mixed message, and often also wrong.
What the user with should do beyond knowing the tool is not exhaustive is up to the user.

What is "new firmware" if not firmware upgrades?